Resources/Prototypes/_RMC14/Entities/Mobs/NPCs/animals.yml

```diff
@@ -18,12 +24,7 @@
   components:
   - type: GhostRole
     prob: 0
-  - type: Respirator
-    damage: {}
-    damageRecovery: {}
-  - type: Barotrauma
-    damage: {}
-
+  - type: PressureImmunity
```

Comment:
You shouldn't need this since it already inherits from RMCSmallHost
```suggestion
```

Reply:
Thanks.